rasa/core/training/interactive.py

```diff
@@ -579,6 +579,20 @@ def _slot_history(tracker_dump: Dict[Text, Any]) -> List[Text]:
     return slot_strings
 
 
+async def _retry_on_error(
+    func: Callable, export_path: Text, *args: Any, **kwargs: Any
+) -> None:
+    while True:
+        try:
+            return func(export_path, *args, **kwargs)
+        except OSError as e:
+            answer = questionary.confirm(
+                f"Failed to export '{export_path}': {e}. Please make sure 'rasa' has read and write access to this file."
```

Comment:
why should actually ask a question. Otherwise it might be confusing for users what they should answer